### DISCUSSION TITLE
NI 2024 Beta Release 2

### DISCUSSION THREAD

**Discussion by jhershbe** [wpilibsuite/2024Beta#46] [Announcements]
Beta 2 of game tools

From internal installer version b96

Includes image version 2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/wpilibsuite/2024Beta/releases/tag/ni-beta-2'>NI 2024 Beta Release 2</a>.</em>

**Comment by chauser** [comment]
Is there a change log/bug fix list for this release?
